public/js/presenter.js

```diff
@@ -300,6 +284,44 @@ function openNotes()
   }
 }
 
+function blankStyledWindow(title, dimensions, zoom) {
+  // yes, the explicit address is needed. Because Chrome.
+  newWindow = window.open('about:blank','', dimensions);
+
+//  $(newWindow.document).ready(function() {
+//  setTimeout(function() {
+  var wtfFirefox = function() {
+    newWindow.document.title = title;
+
+    $('<base/>',{ "href": window.location.origin }).appendTo($(newWindow.document.head));
+    $('link[rel="stylesheet"]').each(function() {
+      $(newWindow.document.head).append($(this).clone());
+    });
+    $(newWindow.document.head).append('<style type="text/css">body { margin: 0.5em; }</style>');
+
+    if(zoom) {
+      // TODO: This will need to be updated to not be terrible on Firefox
+      var style = '<style type="text/css">            \
```

Comment:
Since I know you're going to wig about inline styles, it's because this is dynamically generated.